qchem/pennylane_qchem/qchem.py

```diff
@@ -71,9 +71,9 @@ def read_structure(filepath, outpath="."):
         list: for each atomic species, a list containing the symbol and the Cartesian coordinates
     """
 
-    babel_error_message = (
+    obabel_error_message = (
         "Open Babel converter not found:\n"
-        "Try: 'sudo apt install babel' "
+        "Try: 'sudo apt install openbabel' "
         "or download it from http://openbabel.org/wiki/Main_Page \n"
         "and make sure you add it to the PATH environment variable."
```

Comment:
You could also add the conda install command here for people using anaconda (`conda install openbabel --conda-forge` I think)

Reply:
this: `conda install -c conda-forge openbabel`

Reply:
Apparently it seems apt-get only has version 2.3 where conda installs version 3.0. Which one should we treat as default?

Reply:
I've run the test suite using both the conda version and the apt version, and have noticed no difference --- @agran2018?

Reply:
For our purposes I think they are functionally the same. Based on their release history, there's big overhaul after 2.4, so perhaps worth reminding users about this?
https://github.com/openbabel/openbabel/releases

Reply:
I would suggest to provide users with the largest flexibility. Open Babel can be installed, either via command line or using package manager, in most Linux distributions and ``conda`` alternative is also cross-platform. On the other hand, to inter-convert molecular structure files, there should not be any difference between different versions, this is the simplest and core functionality of Open Babel. thoughts?

Reply:
I tend to agree --- since we are only using a very small subset of openbabel (just the conversions), _if_ the conversion API and features are identical between 2.3 and 3.0, I think keeping the information short and concise is better.

Reply:
Yes, I agree.

Reply:
I have added the command lines for most popular Linux distros and included the conda install command.